src/utils/index.js

```diff
@@ -3,7 +3,7 @@ import html2canvas from 'html2canvas';
 import * as jsPDF from 'jspdf';
 
 const move = (array, element, delta) => {
-  const index = array.indexOf(element);
```

Comment:
indexOf is good for primitive types. we are dealing with objects so I changed it to use findIndex and use id as the key to match items